src/panels/lovelace/cards/hui-sensor-card.js

```diff
@@ -144,8 +140,8 @@ class HuiSensorCard extends EventsMixin(LitElement) {
 
   _calcPoints(history, hours, width, detail = 1) {
     const coords = [];
-    const margin = this._config.line_width;
-    const height = this._config.height - margin * 4;
+    const margin = 5;
+    const height = 100 - margin * 4;
```

Comment:
If it is no longer a variable we don't have to calculate it?
```suggestion
    const height = 380;
```

Reply:
Good Call. I didnt even see that. Thank you